pilot/helpers/agents/Developer.py

```diff
@@ -69,8 +80,9 @@ def implement_task(self, i, development_task=None):
             "array_of_objects_to_string": array_of_objects_to_string,  # TODO check why is this here
             "directory_tree": self.project.get_directory_tree(True),
             "current_task_index": i,
-            "development_tasks": self.project.development_plan,
+            "development_tasks": self.project.feature_development_plan if self.project.finished else self.project.development_plan,
```

Comment:
Can `project.development_plan` and `project.feature_development_plan` be the same field?

Reply:
yes it can, i was using project.development_plan while making feature but then i removed it. Will update this